### PR TITLE
CSS homepage

### PR DESCRIPTION
**Pull request by Theweeksad**
i added links of the css homepage

### CODE REVIEW

**Review by francoo-trainer — APPROVED** [review]

File: CSS/Homepage.css [hunk 1]
```diff
@@ -1,12 +1,7 @@
-.header {
-    display: flex;
-    flex-direction: column;
-}
-
 .header__title {
     display: flex;
     justify-content: center;
-    margin: 50px;
+    margin: 80px 0px 40px 0px;
```
Comment (johanmendezb) [inline CSS/Homepage.css:4]: Please Avoid margin top

File: CSS/Homepage.css [hunk 2]
```diff
@@ -16,13 +11,12 @@
 
 .section {
     margin: 50px 50px;
-    width: 600px;
-    height: 450px;
+    height: 280px;
```
Comment (johanmendezb) [inline CSS/Homepage.css:14]: Avoid giving fixed values for section's height.

File: index.html [hunk 3]
```diff
@@ -34,7 +35,8 @@ <h2 class="header__subtitle">Subtitle</h2>
 
       <section>
         <a class="link__question" href="#">START</a>
-      </section>
+      </section
```
Comment (johanmendezb) [inline index.html:38]: You removed the '>' put it back

**Review by johanmendezb — CHANGES_REQUESTED** [review]
Please review the requested changes

File: CSS/Homepage.css (hunk 1 above)
Comment (johanmendezb) [reply CSS/Homepage.css:4]: unless is the page/main container margins

Comment (Theweeksad) [reply CSS/Homepage.css:4]: fix the margins

File: CSS/Homepage.css (hunk 2 above)
Comment (Theweeksad) [reply CSS/Homepage.css:14]: change the heigth to a margin

File: index.html (hunk 3 above)
Comment (Theweeksad) [reply index.html:38]: I have already closed the section tag with ">"

**Comment by ripsamuel** [comment]
good job
please attacht screenshot of the layaout

**Review by johanmendezb — APPROVED** [review]

**Pull request merged**